### PR TITLE
Handle client disconnect

### PR DESCRIPTION
**Pull request by pjagielski**

### CODE REVIEW

File: nifi-ignite-extensions/nifi-ignite-processors/src/main/java/pl/touk/nifi/processors/PutIgniteRecord.java [hunk 1]
```diff
@@ -123,18 +140,21 @@ public final void closeIgniteDataStreamerAndCache() {
...
             return;
         }
 
-        getLogger().info("Creating Ignite Datastreamer");
+        getLogger().info("Creating Ignite DataStreamer");
         try {
             int perNodeParallelOperations = context.getProperty(DATA_STREAMER_PER_NODE_PARALLEL_OPERATIONS).asInteger();
             int perNodeBufferSize = context.getProperty(DATA_STREAMER_PER_NODE_BUFFER_SIZE).asInteger();
-            int autoFlushFrequency = context.getProperty(DATA_STREAMER_AUTO_FLUSH_FREQUENCY).asInteger();
+            int autoFlushFrequency = context.getProperty(DATA_STREAMER_MAX_RETRIES_ON_FAILURE).asInteger();
```
Comment (nadberezny) [inline nifi-ignite-extensions/nifi-ignite-processors/src/main/java/pl/touk/nifi/processors/PutIgniteRecord.java]: I guess, this change was not intened

File: nifi-ignite-extensions/nifi-ignite-processors/src/main/java/pl/touk/nifi/processors/PutIgniteRecord.java [hunk 2]
```diff
@@ -176,13 +196,24 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
...
 
-        if (!cacheItems.isEmpty()) {
+        int retries = 0;
+
+        while (retries < maxRetries && !cacheItems.isEmpty()) {
             try {
                 IgniteFuture<?> futures = igniteDataStreamer.addData(cacheItems);
                 Object result = futures.get();
                 getLogger().trace("Result {} of addData", new Object[]{result});
+                cacheItems.clear();
```
Comment (nadberezny) [inline nifi-ignite-extensions/nifi-ignite-processors/src/main/java/pl/touk/nifi/processors/PutIgniteRecord.java:206]: :)

**Review by nadberezny — CHANGES_REQUESTED** [review]
:1st_place_medal: 
LGTM
just one line to fix

File: nifi-ignite-extensions/nifi-ignite-processors/src/main/java/pl/touk/nifi/processors/PutIgniteRecord.java (hunk 1 above)
Comment (pjagielski) [reply nifi-ignite-extensions/nifi-ignite-processors/src/main/java/pl/touk/nifi/processors/PutIgniteRecord.java]: good catch!

**Review by nadberezny — APPROVED** [review]
:+1:

**Pull request merged**